src/DynamoCoreWpf/ViewModels/RunSettingsViewModel.cs

```diff
@@ -81,12 +81,12 @@ public RunTypeItem(RunType runType)
     /// notifications as those notifications are raised when the value is set on the
     /// model.
     /// </summary>
-    public class RunSettingsViewModel : NotificationObject
+    public class RunSettingsViewModel : ViewModelBase
```

Comment:
this is an API break.

Reply:
or - could be, we need to be very careful changing the base class of a public class.

Reply:
How do we find out? ViewModelBase implements `NotificationObject` and `IDisposal` so I believe doing this will not remove any public property, that's actually why I thought it is not API breaking. Same reason below.